.github/workflows/check_schema_version.sh

```diff
@@ -20,7 +26,17 @@ function get_all_schemaVersion_defining_files () {
 for testFile in $(get_all_schemaVersion_defining_files)
 do
     schemaVersion=$(get_schema_version "$testFile")
-    if ! ajvCheck=$(ajv -s "source/unified-test-format/schema-$schemaVersion.json" -d "$testFile"); then
+    jsonSchemaURL=$(get_json_schema_url "$testFile")
+    if [ "$jsonSchemaURL" = "https://json-schema.org/draft/2019-09/schema#" ]; then
+      spec="draft2019"
+    elif [ "$jsonSchemaURL" = "http://json-schema.org/draft-07/schema#" ]; then
+      spec="draft7"
+    else
+      echo "Do not know how to validate $jsonSchemaURL"
+      exit 1
+    fi
+
+    if ! ajvCheck=$(ajv --spec="$spec" -s "source/unified-test-format/schema-$schemaVersion.json" -d "$testFile"); then
```

Comment:
Noted that this pertains to https://ajv.js.org/packages/ajv-cli.html#json-schema-language-and-version

Did something in this PR require introduction of the `--spec` option for `ajv-cli`? It looks like the schema URL was changed in https://github.com/mongodb/specifications/commit/db693517de561465621b474558ded99eaf38757d when `deprecated` options were introduced.